tools/docker-dev/Dockerfile

```diff
@@ -26,40 +27,44 @@ vim \
 
 # Create user and allow sudo without password.
 RUN addgroup --quiet --gid $gid $group \
-&& adduser --quiet --disabled-password --gecos ',,,,' --uid $uid --ingroup $group $user \
+&& adduser --quiet --disabled-password --gecos ",,,," --uid $uid --ingroup $group $user \
 && echo "${user} ALL=(ALL:ALL) NOPASSWD: ALL" > /etc/sudoers.d/$user
 
 # Install Docker CLI.
 COPY --from=docker /usr/local/bin/docker /usr/local/bin/docker
 
 # Fix the Docker socket access rights at login time to allow non-root access.
-RUN echo 'sudo chmod o+rw /var/run/docker.sock' >> /home/${user}/.bashrc
+RUN echo "sudo chmod o+rw /var/run/docker.sock" >> /home/${user}/.bashrc
 
 # Install Go.
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 RUN curl -s -Lo - https://dl.google.com/go/go${goversion}.linux-amd64.tar.gz | tar -C /usr/local -xzf - \
-&& echo '# Go environment.' >> /home/${user}/.bashrc \
-&& echo 'export GOROOT=/usr/local/go' >> /home/${user}/.bashrc \
-&& echo 'export GOPATH=~/go' >> /home/${user}/.bashrc \
-&& echo 'export PATH=$GOROOT/bin:$GOPATH/out/linux_amd64/release:$GOPATH/bin:$PATH' >> /home/${user}/.bashrc \
-&& echo 'export GO111MODULE=on' >> /home/${user}/.bashrc \
+&& echo "# Go environment." >> /home/${user}/.bashrc \
+&& echo "export GOROOT=/usr/local/go" >> /home/${user}/.bashrc \
+&& echo "export GOPATH=~/go" >> /home/${user}/.bashrc \
+&& echo "export PATH=\$GOROOT/bin:\$GOPATH/out/linux_amd64/release:\$GOPATH/bin:\$PATH" >> /home/${user}/.bashrc \
+&& echo "export GO111MODULE=on" >> /home/${user}/.bashrc \
 && mkdir -p /home/${user}/go
 
 # Install KIND 0.3.0.
 # Cf. https://github.com/kubernetes-sigs/kind
 RUN GO111MODULE="on" GOROOT=/usr/local/go GOPATH=/home/${user}/go /usr/local/go/bin/go get sigs.k8s.io/kind@v0.3.0
 
 # Install Helm's latest release.
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
```

Comment:
I believe this `SHELL` command only needs to be specified once near the top of each Dockerfile. This one could probably be removed?